h/views.py

```diff
@@ -39,9 +39,14 @@ def annotation(context, request):
     alternate = request.resource_url(request.root, 'api', 'annotations',
                                      context['id'])
 
+    description = context.get('text')
+    if len(description) > 200:
+        description = description[:200] + '...'
```

Comment:
The reason I wouldn't add ellipses is because the client that displays these (Facebook, etc) might do something like this. We can go over the character limit we shoot for, too, and they'll truncate accordingly and apply whatever style they think is helpful.